install.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 
+set -exuo pipefail
```

Comment:
`set -x` here could be really noisy?

```suggestion
set -exuo pipefail
```

But if you've checked the output, and think we should keep it, lets do.

Reply:
I added it because when I ran the script locally it was hanging on the `gem install bundler` step but I couldn't tell. 

I do plan on running this install script as part of a `server-templates` script to install and launch lighthouse-docker. Do you think it will be too noisy in those logs?

Reply:
Oh, that makes sense. Probably fine. This script is simple enough that it wont generate too much noise. And more explicit logs are usually better than less. Disregard.